static/templates/lightbox_overlay.hbs

```diff
@@ -4,16 +4,12 @@
             <div class="title"></div>
             <div class="user"></div>
         </div>
-        <div class="exit" aria-label="{{t 'Close' }}"><span aria-hidden="true">x</span></div>
         <div class="image-actions">
-            <div class="lightbox-canvas-trigger">
-                <div class="title">{{t "Pan & zoom" }}</div>
-                <div class="status" data-disabled="{{t 'Disabled' }}" data-enabled="{{t 'Enabled' }}"></div>
-            </div>
+            <a class="button small lightbox-zoom-reset disabled">{{t "Reset Image" }}</a>
```

Comment:
I believe this requires a new translation. If we are happy with this wording, I can go ahead and try and do that.

Reply:
Translations are managed separately in Transifex, so no need to make any changes other than to strings; see
https://zulip.readthedocs.io/en/latest/translating/internationalization.html

But we do need to use "Sentence case" as we do everywhere in Zulip; this will fail lint for that reason.

Reply:
It did fail linting and I fixed it last night, this comment is on the outdated version of that line.